fabtests/functional/av_xfer.c

```diff
@@ -233,8 +233,7 @@ int main(int argc, char **argv)
 	if (optind < argc)
 		opts.dst_addr = argv[optind];
 
-	hints->caps = hints->ep_attr->type == FI_EP_RDM ?
-		      FI_TAGGED : FI_MSG;
+	hints->caps = FI_MSG;
```

Comment:
This should be a separate PR, at least we shouldn't mention efa-direct as the motivation of this common change

Reply:
Even if the test doesn't require tag matching, it's better to allow `FI_MSG` and `FI_TAGGED` toggling via `-o`

Reply:
moved the fabtests commit to https://github.com/ofiwg/libfabric/pull/10828